.github/workflows/go.yml

```diff
@@ -10,51 +10,29 @@ on:
     branches: [ "main" ]
 
 jobs:
-
-  build-for-linux:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: Set up Go
-      uses: actions/setup-go@v4
-      with:
-        go-version: 1.22.x
-
-    - name: Install Protoc
-      uses: arduino/setup-protoc@v3
-      with:
-        version: "23.x"
-
-    - name: Set up protoc-gen-go
-      run: go install google.golang.org/protobuf/cmd/protoc-gen-go
-
-    - name: Build
-      run: make
-
-    - name: Test
-      run: make test
-
-  build-for-windows:
-    runs-on: windows-latest
+  tests:
+    stragegy:
```

Comment:
Actually I think there's a typo here. This should be "strategy" right? Currently we get an error when running the github actions: https://github.com/google/osv-scalibr/actions/runs/11707281609

Fixing this in https://github.com/google/osv-scalibr/pull/271

Reply:
Well that's very embarrassing - thanks for the catch

(Shouldn't happen in future now that my prs will actually trigger workflow runs 😁)